frontend/libs/erxes-ui/src/modules/record-field/components/TextField.tsx

```
@@ -20,46 +20,72 @@ export const TextField = React.forwardRef<
 
-  const handleAction = () => {
-    if (editingValue === value) return;
-    onValueChange && onValueChange(editingValue);
-  };
+    const handleAction = (e?: React.FormEvent) => {
+      e?.preventDefault();
+      if (editingValue === value) {
+        setIsOpen(false);
+        return;
+      }
+      onSave && onSave(editingValue);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)